### DISCUSSION TITLE
aarch64-linux-android-ar: */*.o: Invalid argument

### DISCUSSION THREAD

**Discussion by 1079107009** [Tencent/libpag#1760] [Q&A] [answered]
![image](https://github.com/Tencent/libpag/assets/24666023/8fee34bc-43a5-4411-a78c-0d0d80efc922)
想问一下这行命令的作用是什么，在windows系统下报错，请问有没有解决办法？

**Comment by domchen** [comment, marked as answer]
你不能直接去改这个脚本，按照readme正常配置好开发环境就不会报错了。Visual Sutdio要看文档最少安装指定的C++相关模块才有对应编译工具。另外Windows平台只能编译Windows版本，Android是只能在mac平台编译。我们没有在Windows下开发Android的需求。

**Comment by domchen** [comment]
这个脚本运行的时候，还要全局设置工具链目录的环境变量，所以你单独改个绝对路径指向某个命令行这是没用的。正常安装指定版本的VS，让脚本自己找到正确目录。
